ktor-io/jvm/src/io/ktor/utils/io/ByteReadChannelJVM.kt

```diff
@@ -236,6 +252,9 @@ public actual interface ByteReadChannel {
         max: Long
     ): Long
 
+    @ExperimentalIoApi
```

Comment:
This annotation is deprecated, please consider introducing an issue of why this API is experimental and create a new annotation for that.